src/OpenLoco/Map/TileManager.cpp

```diff
@@ -561,7 +561,11 @@ namespace OpenLoco::Map::TileManager
         {
             for (uint8_t xOffset = 0; xOffset < 11; xOffset++)
             {
-                auto tile = get(initialTilePos + Map::TilePos2(xOffset, yOffset));
+                auto tilePos = initialTilePos + Map::TilePos2(xOffset, yOffset);
+                if (tilePos.x < 0 || tilePos.y < 0)
```

Comment:
Should also check the max range. Tbh should make a function for validating it will need to use it in lots of places